public/app/plugins/datasource/loki/metricTimeSplit.test.ts

```diff
@@ -0,0 +1,16 @@
+import { getRanges } from './metricTimeSplit';
+
+describe('querySplit', () => {
+  // FIXME add more tests, for cases like too small chunk-sizes etc.
```

Comment:
Would it be to much to add those tests now, so we make sure this implementation is the closest possible to a final one?

Reply:
sure, will do!